staging/cert-manager-setup/templates/certificates.yaml

```
@@ -10,9 +10,12 @@ metadata:
     "helm.sh/hook-weight": "-3"
     "helm.sh/hook-delete-policy": before-hook-creation
 spec:
-  isCA: true
-  commonName: cert-manager
-  duration: {{ .duration | default "87600h" | quote }}
+{{- if .acme }}
+{{ . | toYaml | indent 2 }}
+{{- end }}
```

consider indenting for readability and using nindent
```suggestion
{{- if .acme }}
  {{ . | toYaml | nindent 2 }}
{{- end }}
```

---

this would add a new line, which is not what we want.
`n` for `newline`